hack/ci-deploy.sh

```diff
@@ -38,66 +39,16 @@ kubectl wait --for condition=established --timeout=60s crd/certificates.cert-man
 wait-for-object-creation cert-manager deployment.apps/cert-manager-webhook
 kubectl -n cert-manager rollout status --timeout=5m deployment.apps/cert-manager-webhook
 
-kubectl apply -f"${deploy_dir}"/operator.yaml
-kubectl apply -f"${deploy_dir}"/scylla-operator-member.clusterrole.yaml
+kubectl apply -f"${deploy_dir}"/operator
 
 # Manager needs scylla CRD registered and the webhook running
 kubectl wait --for condition=established crd/scyllaclusters.scylla.scylladb.com
-kubectl -n scylla-operator-system rollout status --timeout=5m statefulset.apps/scylla-operator-controller-manager
+kubectl -n scylla-operator rollout status --timeout=5m deployment.apps/scylla-operator
 
 # ScyllaCluster doesn't support changes and is immutable so we always have to delete/create it.
-kubectl delete --ignore-not-found=true --grace-period=0 --cascade=foreground -f"${deploy_dir}"/manager.yaml
-kubectl create -f"${deploy_dir}"/manager.yaml
-object=$( kubectl -n scylla-manager-system get scyllacluster/scylla-manager-cluster -o json | jq '.metadata.uid = "" | .metadata.resourceVersion = "" | .metadata.generation = 0 | .spec.datacenter.racks[0].resources = {"requests":{"cpu":"10m","memory":"100Mi"},"limits":{"cpu":"200m","memory":"500Mi"}}')
-kubectl delete --grace-period=0 --cascade=foreground -f <( echo "${object}" )
-kubectl delete --ignore-not-found=true pvc/data-scylla-manager-cluster-manager-dc-manager-rack-0
-kubectl create -f <( echo "${object}" )
-kubectl patch -n scylla-manager-system deployment/scylla-manager-scylla-manager --type=strategic -p='{"spec":{"template":{"spec":{"containers":[{"name":"scylla-manager","resources":{"requests":{"cpu":"10m","memory":"100Mi"},"limits":null}}]}}}}'
-# initContainers patch is broken
-wait-for-object-creation scylla-manager-system statefulset.apps/scylla-manager-cluster-manager-dc-manager-rack
-kubectl -n scylla-manager-system get statefulset.apps/scylla-manager-cluster-manager-dc-manager-rack -o json | jq '.spec.template.spec.initContainers[0].resources = {"requests":{"cpu":"10m","memory":"100Mi"}} | .metadata.managedFields = null' | kubectl -n scylla-manager-system apply --server-side --force-conflicts -f -
+kubectl delete --ignore-not-found=true --grace-period=0 --cascade=foreground -f"${deploy_dir}"/manager
```

Comment:
we can use just apply, I think the delete was here because I had to modify the resource with jq and it created a conflict